tests/dummy/config/environment.js

```diff
@@ -1,10 +1,9 @@
 /* eslint-env node */
-'use strict';
 
 module.exports = function(environment) {
-  let ENV = {
+  var ENV = {
     modulePrefix: 'dummy',
-    environment,
+    environment: environment,
```

Comment:
yah, this happened as part of the `ember init` phase of upgrading ember-cli. It was one of those things where I figured someone must have changed it for a reason...